tmf/org.eclipse.tracecompass.tmf.ui/src/org/eclipse/tracecompass/tmf/ui/views/histogram/HistogramView.java

```diff
@@ -610,6 +610,11 @@ private void setNewRange(long startTime, long duration) {
             realStart = fTraceStartTime;
         }
 
+        if (duration == 1) {
+            fTimeSpanControl.setValue(fWindowSpan);
```

Comment:
Do we need this?

Reply:
This sets the value in the text box to its previous value. 
I.e. if we don't use that, we will still have 1ns still written in the text box even if the histogram isn't updated.

Reply:
Indeed, I misread duration... oops :)